torch/csrc/jit/script/compiler.cpp

```diff
@@ -563,6 +564,55 @@ struct to_ir {
     ConstantPooling(to_clean);
   }
 
+void runInline(Block* block) {
```

Comment:
Separate file please. compiler.cpp is long enough. This will need to be detangled from things like the environment. Otherwise the PR is not really accomplishing its purpose.